.circleci/config.yml

```diff
@@ -18,18 +25,35 @@ jobs:
 
   build:
     docker:
-      - image: circlecidx/circleci-cli:0.1.228
+      - image: circleci/circleci-cli:0.1.1418
     steps:
       - checkout
       - run: "echo -e \"token: placeholder\nverbose: false > ~/.circleci/cli.yml\""
-      - run: "for ORB in src/*; do circleci orb validate --path $ORB/orb.yml; done"
+      - run: "sh scripts/validate-orbs.sh"
 
-  release:
-    machine: true
+  dev-release:
+    docker:
+      - image: circleci/circleci-cli:0.1.1418
     steps:
       - checkout
-      ## install the cirlceci cli (or get it into a Docker image we use in the executor)
+      - run: "sh scripts/dev-release.sh"
+
+  trigger-downstream:
+    machine: true
+    steps:
+      - run: |
```

Comment:
It would be a time saver to add a link to the test build here: https://circleci.com/gh/circleci/aws-code-deploy-orb-test